README.md

```diff
@@ -166,7 +166,7 @@ You can visite the online version at : <a href="https://grandi0z.github.io/#">gr
 👤 **Author1**
 
 - GitHub: [@Grandi0z](https://github.com/Grandi0z)
-- Twitter: [@wa_inje](https://twitter.com/wa_inje)
+- Twitter: [@wa_inje](https://twitter.com/Joseph_LSK)
```

Comment:
- [ ] Hi👋, nice job updating your social media links, but please remember you also need to add to your readme file the link to your deployed page using GitHub pages.